oak_abi/src/lib.rs

```diff
@@ -213,6 +213,10 @@ extern "C" {
 
     /// Creates a new Node instance running code identified by a serialized [`NodeConfiguration`].
     ///
+    /// The name of the new node is provided in the memory area given by `name_buf` and `name_len`.
+    /// The name does not have to be unique and can be empty. It is used in logs to help identify
+    /// nodes for debugging purposes.
```

Comment:
Since it is only used in debug, should we consider having a separate `node_create` function without a name?
Or it will be too much for ABI

Reply:
I think having multiple functions doing the same thing is will make the ABI unnecessarily comple. Also, se discussion in https://github.com/project-oak/oak/issues/1615#issuecomment-716663914